README.md

```diff
@@ -188,6 +188,10 @@ $ ganache
 This will create a local blockchain and also some test accounts that are used throughout our tests.
 Make sure that `ganache` is running on its default port `8546`. To change the port in test cases locate `LocalTestCase.swift` and modify the static `url` variable.
 
+### Before you commit
+
+We are using [pre-commit](https://pre-commit.com) to run validations locally before a commit is created. Please, install pre-commit and run `pre-commit install` from project's root directory. After that before every commit git hook will run and execute `codespell`, `swiftlint` and other checks.
```

Comment:
Maybe some tip about how to install pre-comit or link to doc, like website install methods 
https://pre-commit.com/#install
ie
```
Please, install pre-commit and run `pre-commit install` from project's root directory, check out pre-commit [website](https://pre-commit.com/#install) for more info
```

Reply:
But there is already a link: `We are using [pre-commit](https://pre-commit.com)`.
And on the front page you have the installation.

Reply:
ok, I understand. I suppose it is enough